docs/EN/Configuration/Devices.md

```diff
@@ -2,12 +2,12 @@
 Adding devices to Trio is easy. You can only have one pump and one CGM active simultaneously. If you want to switch to another pump or CGM, delete the current device first. When you select a device, you will be guided through its setup with an on-screen guide.
 
 ## Pump
-This section asks you to select your insulin delivery system. For more information on supported pumps, go to [Getting Started](../Getting-Started/pump.md). Omnipod Eros and older Medtronic devices don't support Bluetooth. These devices require a Rileylink, OrangeLink, or EmaLink to translate communication between your phone's Bluetooth signals and the pump's radio communication signals.
+This section asks you to select your insulin delivery system. For more information on supported pumps, go to [Getting Started](Getting-Started/pump.md). Omnipod Eros and older Medtronic devices don't support Bluetooth. These devices require a Rileylink, OrangeLink, or EmaLink to translate communication between your phone's Bluetooth signals and the pump's radio communication signals.
```

Comment:
This link was correct before. Perhaps your reviewers were looking at an outdated version of the docs.
You must go up one level from Configuration and then select Getting-Started, so `(../Getting-Started/pump.md)` is correct.

Reply:
I thought so as well, but the link didn't work with the "../", but it did when I removed it. I will add it back and see if the error persists.